stubs/default/resources/css/app.css

```
@@ -1,3 +1,13 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import 'tailwindcss';
+
+@source './vendor/laravel/framework/src/Illuminate/Pagination/resources/views/*.blade.php';
+@source './storage/framework/views/*.php';
+@source "./resources/views/**/*.blade.php";
+@source "./resources/js/**/*.vue";
+
```

```suggestion
@source '../../vendor/laravel/framework/src/Illuminate/Pagination/resources/views/*.blade.php';
@source '../../storage/framework/views/*.php';
@source "../views/**/*.blade.php";
@source "../js/**/*.vue";
```

The `app.css` lies in `resources/css` therefor the paths need to adjusted.

---

Thanks